radio/src/functions.cpp

```diff
@@ -507,14 +507,45 @@ void evalFunctions()
 #else
           case FUNC_PLAY_SOUND:
           {
+#if defined(AUDIO)
             tmr10ms_t tmr10ms = get_tmr10ms();
             uint8_t repeatParam = CFN_PLAY_REPEAT(cfn);
-            if (!functionsContext.lastFunctionTime[i] || (repeatParam && (signed)(tmr10ms-functionsContext.lastFunctionTime[i])>=1000*repeatParam)) {
+            if (!functionsContext.lastFunctionTime[i] || (repeatParam && (signed)(tmr10ms-functionsContext.lastFunctionTime[i])>=100*repeatParam)) {
```

Comment:
I don't think it was sorry. Thanks for keeping such a close eye on it.

Fixed in my working copy.